css/style-contacts.css

```diff
@@ -0,0 +1,220 @@
+
+
+.contact { 
+  max-width: 1500px;
+  min-height: 866px;
+  background: url(../image/2ce-bakg.png) no-repeat;
+  margin: 0 auto;
+}
+
+/*headr*/
+.contact-header { 
+  padding-top: 101px;
+}
+
+.contact-header__title { 
+  color: #262626;
+  font-family: "Raleway", sans-serif;
+  font-size: 90px;
+  font-weight: 700;
+  line-height: 25px;
+  text-align: center;
+}
+
+.contact-header__text { 
+  width: 350px;
+  color: #262626;
+  font-family: "Raleway", sans-serif;
+  font-size: 17px;
+  font-weight: 300;
+  line-height: 25px;
+  text-align: center;
+  margin: 68px auto;
+}
+
+/*main*/
+.main { 
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.main-form { 
+  display: flex;
+  flex-direction: column;
+}
+
+.main-form__input-first{
+  width: 500px;
+  height: 50px;
+  background-color: #00c4c4;
+  opacity: 0.5;
+  padding-left: 20px;
+  margin-bottom: 15px;
+  font-weight: 300;
+}
+
+.main-form__input-second{
+  width: 500px;
+  height: 50px;
+  background-color: #00c4c4;
+  opacity: 0.5;
+  padding-left: 20px;
+  margin-bottom: 15px;
+  font-weight: 300;
+}
+
+.main-form__input-three{
+  width: 500px;
+  height: 50px;
+  background-color: #00c4c4;
+  opacity: 0.5;
+  padding-left: 20px;
+  margin-bottom: 15px;
+  font-weight: 300;
+}
+
+.main-form__input-four {
+  width: 500px;
+  height: 250px;
+  background-color: #00c4c4;
+  opacity: 0.5;
+  padding-left: 20px;
+  margin-bottom: 68px;
+  font-weight: 300;
+}
+
+.main-form label input::placeholder{
```

Comment:
Be careful, this rule increases specificity which can lead to unexpected behavior